src/__test__/test-utils/experimentSettings.mock.js

```diff
@@ -14,6 +14,16 @@ const sampleifiedConfig = (sampleIds, configToReplicate) => {
 };
 
 const generateProcessingConfigMock = (sampleIds) => ({
+  classifier: {
```

Comment:
what about the prefiltered parameter?

Reply:
oh right. If `enabled` is true, then `prefiltered` is false.